src/components/Header.tsx

```diff
@@ -118,7 +118,11 @@ const Header = (props: BoxProps) => {
                 placeholder="blur"
               />
             </Box>
-            <Heading fontSize={{ base: 'md', md: 'xl', lg: '4xl' }}>
+            <Heading
+              letterSpacing="tight"
```

Comment:
The _Inter_ font isn't designed to do work as a "display font" that used with big font sizes. I'm tightening the letter-spacing to condense the font, so it looks a bit better.